controllers/terraform_controller.go

```diff
@@ -915,6 +945,30 @@ terraform {
 	return terraform, tfInstance, tmpDir, nil
 }
 
+func (r *TerraformReconciler) createRunnerFileMapping(ctx context.Context, terraform infrav1.Terraform) ([]*runner.FileMapping, error) {
+	var runnerFileMappingList []*runner.FileMapping
+
+	for _, fileMapping := range terraform.Spec.RunnerPodTemplate.Spec.FileMappings {
+		var runnerFileMapping runner.FileMapping
```

Comment:
Why do you need this fwd declaration? 
You can init the struct in-place when appending